### ISSUE TITLE
explorer.exe crashes on system startup

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#84]
```
Which program triggers the problem? Specify the filename if possible.
at a guess, gdipp_svc_64.exe 

What steps will reproduce the problem?
1.Install 0.7.6 as service
2.Startup computer with x64 set to automatic
3.Explorer.exe crashes

What is the expected output? What do you see instead?
Explorer.exe... not to crash :)

What version of gdipp (including renderer)? On which operating system?
gdipp 0.7.6 freetype renderer, Windows 7 x64

Please provide any additional information below. Screenshots always help.
Setting service to manual, then starting after startup has completed everything 
else does _not_ cause a crash of explorer.exe.

```

Original issue reported on code.google.com by `Kawi...@gmail.com` on 3 Jul 2010 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
What if you set the service' startup type to Automatic (Delayed Start)?
```

Original comment by `crendk...@gmail.com` on 4 Jul 2010 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
Tried just now, seems to do the trick nicely :)
```

Original comment by `Kawi...@gmail.com` on 4 Jul 2010 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
Other users also reported that 0.7.6 may crash explorer randomly. There was 
some flaw in the design of the font manager when multiple threads are 
competing. I have corrected the problem in 0.8.x and I personally have not 
encountered any explorer crash. Would you mind try 0.8.2 some time? Thanks.
```

Original comment by `crendk...@gmail.com` on 4 Jul 2010 at 1:16
- Changed state: **Done**
